### ISSUE TITLE
Scroll containers don't fill the screen

### ISSUE THREAD

**Issue by joebochill** [brightlayer-ui/react-native-workflows#75]
#### Describe the bug
The ScrollViews on many screens are nested within a View with a max width of 600, which means you can only scroll the screen by dragging your finger in the middle 600px wide column of the screen.

#### What is the expected behavior?
The page should be scrollable by dragging the finger anywhere on the page.
	
#### Anything else to add?
We should be able to let the ScrollView fill the entire page and set a max width on its contents instead.

**Comment by huayunh** [comment]
This is for the tablet view

 not a high priority for the next release

**Comment by joebochill** [comment]
I took a brief look at this and it looks like using the `contentContainerStyle` prop and setting it to `{width: '100%', maxWidth: 600, alignSelf: 'center'}` does the trick. This allows us to clean up a bunch of the extra/unnecessary View wrappers. I did this in two spots already as part of another PR, but didn't do all of the screens because it's a pretty touchy change and didn't want to complicate my other bug fix review.

But it should be pretty straightforward to do this.
